aws/policy/security-services.yaml

```
@@ -15,13 +15,15 @@ Statement:
     Resource:
       - arn:aws:iam::{{ aws_account_id }}:role/ansible-test-sts-*
       - arn:aws:iam::{{ aws_account_id }}:instance-profile/ansible-test-sts-*
```

(Oh, oops, this is still needed for ec2_instance tests though if they switch from using ansible-test-sts. Might as well add that now.)
```suggestion
       - arn:aws:iam::{{ aws_account_id }}:instance-profile/ansible-test-sts-*
       - arn:aws:iam::{{ aws_account_id }}:instance-profile/ansible-test-*
```
Let me know when you want to deploy to prod, looks good to me besides that.

---

@s-hertel  Good to go - want me to do that or do you already have things in flight?

---

@jillr Go for it!